Resources/Prototypes/Reagents/botany.yml

```diff
@@ -145,6 +145,17 @@
             Asphyxiation: 1
             Burn: 2
             Poison: 1
+    Medicine:
+      effects:
+      - !type:Polymorph
+        prototype: TreeMorph
+        conditions:
+          - !type:OrganType
+            type: Plant
+          - !type:ReagentThreshold
+            min: 50
```

Comment:
50 feels really low for what's essentially a permakill

Reply:
yeah an easy way i thought of to kill diona with it is first give a 25u robust pill tell em its robust for brute then another one but its labelled 10u derma for burns boom round removed before they can even realise you trolled

Reply:
Moved to 80

Reply:
> yeah an easy way i thought of to kill diona with it is first give a 25u robust pill tell em its robust for brute then another one but its labelled 10u derma for burns boom round removed before they can even realise you trolled

1. Pill cap on upstream is 20u.
2. You'd have to eat 60-ish units for you to have over 50u of robust in you at once, since you metabolize it.

Actually come to think of it I want to revert it back to 50-60u since you can pkill only with a syringe at that point. You wont get to make them to be a tree just off the pills.